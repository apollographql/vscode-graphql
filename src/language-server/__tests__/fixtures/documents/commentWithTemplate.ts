// prettier-ignore
{
const _1 = /* GraphQL */`query {test}`;
const _2 =
/* GraphQL */`
query {
  test
}
`;
}

/* GraphQL */ `query {test}`;
/*GraphQL*/ `query {test}`;
/** GraphQL */ `query {test}`;
/**
 * GraphQL
 */ `query {test}`;

/* GraphQL */
`query {test}`;

const _3 = /* GraphQL */ `
  query {
    test
  }
`;

/* graphql */
`query {test}`;

/* gql */
`query {test}`;
