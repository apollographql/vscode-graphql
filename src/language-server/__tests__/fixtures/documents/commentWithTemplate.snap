>// prettier-ignore
#^^^^^^^^^^^^^^^^^^^ inline.graphql
>{
#^^ inline.graphql
>const _1 = /* GraphQL */`query {test}`;
#^^^^^^^^^^^ inline.graphql
#           ^^^^^^^^^^^^^ inline.graphql apollo.commentWithTemplate comment.block.js
#                        ^ inline.graphql apollo.commentWithTemplate punctuation.definition.string.template.begin.js
#                         ^^^^^ inline.graphql apollo.commentWithTemplate meta.embedded.block.graphql keyword.operation.graphql
#                              ^ inline.graphql apollo.commentWithTemplate meta.embedded.block.graphql meta.selectionset.graphql
#                               ^ inline.graphql apollo.commentWithTemplate meta.embedded.block.graphql meta.selectionset.graphql punctuation.operation.graphql
#                                ^^^^ inline.graphql apollo.commentWithTemplate meta.embedded.block.graphql meta.selectionset.graphql variable.graphql
#                                    ^ inline.graphql apollo.commentWithTemplate meta.embedded.block.graphql meta.selectionset.graphql punctuation.operation.graphql
#                                     ^ inline.graphql apollo.commentWithTemplate punctuation.definition.string.template.end.js
#                                      ^^ inline.graphql
>const _2 =
#^^^^^^^^^^^ inline.graphql
>/* GraphQL */`
#^^^^^^^^^^^^^ inline.graphql apollo.commentWithTemplate comment.block.js
#             ^ inline.graphql apollo.commentWithTemplate punctuation.definition.string.template.begin.js
>query {
#^^^^^ inline.graphql apollo.commentWithTemplate meta.embedded.block.graphql keyword.operation.graphql
#     ^ inline.graphql apollo.commentWithTemplate meta.embedded.block.graphql meta.selectionset.graphql
#      ^ inline.graphql apollo.commentWithTemplate meta.embedded.block.graphql meta.selectionset.graphql punctuation.operation.graphql
>  test
#^^ inline.graphql apollo.commentWithTemplate meta.embedded.block.graphql meta.selectionset.graphql
#  ^^^^ inline.graphql apollo.commentWithTemplate meta.embedded.block.graphql meta.selectionset.graphql variable.graphql
>}
#^ inline.graphql apollo.commentWithTemplate meta.embedded.block.graphql meta.selectionset.graphql punctuation.operation.graphql
>`;
#^ inline.graphql apollo.commentWithTemplate punctuation.definition.string.template.end.js
# ^^ inline.graphql
>}
#^^ inline.graphql
>
>/* GraphQL */ `query {test}`;
#^^^^^^^^^^^^^ inline.graphql apollo.commentWithTemplate comment.block.js
#             ^ inline.graphql apollo.commentWithTemplate
#              ^ inline.graphql apollo.commentWithTemplate punctuation.definition.string.template.begin.js
#               ^^^^^ inline.graphql apollo.commentWithTemplate meta.embedded.block.graphql keyword.operation.graphql
#                    ^ inline.graphql apollo.commentWithTemplate meta.embedded.block.graphql meta.selectionset.graphql
#                     ^ inline.graphql apollo.commentWithTemplate meta.embedded.block.graphql meta.selectionset.graphql punctuation.operation.graphql
#                      ^^^^ inline.graphql apollo.commentWithTemplate meta.embedded.block.graphql meta.selectionset.graphql variable.graphql
#                          ^ inline.graphql apollo.commentWithTemplate meta.embedded.block.graphql meta.selectionset.graphql punctuation.operation.graphql
#                           ^ inline.graphql apollo.commentWithTemplate punctuation.definition.string.template.end.js
#                            ^^ inline.graphql
>/*GraphQL*/ `query {test}`;
#^^^^^^^^^^^ inline.graphql apollo.commentWithTemplate comment.block.js
#           ^ inline.graphql apollo.commentWithTemplate
#            ^ inline.graphql apollo.commentWithTemplate punctuation.definition.string.template.begin.js
#             ^^^^^ inline.graphql apollo.commentWithTemplate meta.embedded.block.graphql keyword.operation.graphql
#                  ^ inline.graphql apollo.commentWithTemplate meta.embedded.block.graphql meta.selectionset.graphql
#                   ^ inline.graphql apollo.commentWithTemplate meta.embedded.block.graphql meta.selectionset.graphql punctuation.operation.graphql
#                    ^^^^ inline.graphql apollo.commentWithTemplate meta.embedded.block.graphql meta.selectionset.graphql variable.graphql
#                        ^ inline.graphql apollo.commentWithTemplate meta.embedded.block.graphql meta.selectionset.graphql punctuation.operation.graphql
#                         ^ inline.graphql apollo.commentWithTemplate punctuation.definition.string.template.end.js
#                          ^^ inline.graphql
>/** GraphQL */ `query {test}`;
#^^^^^^^^^^^^^^ inline.graphql apollo.commentWithTemplate comment.block.js
#              ^ inline.graphql apollo.commentWithTemplate
#               ^ inline.graphql apollo.commentWithTemplate punctuation.definition.string.template.begin.js
#                ^^^^^ inline.graphql apollo.commentWithTemplate meta.embedded.block.graphql keyword.operation.graphql
#                     ^ inline.graphql apollo.commentWithTemplate meta.embedded.block.graphql meta.selectionset.graphql
#                      ^ inline.graphql apollo.commentWithTemplate meta.embedded.block.graphql meta.selectionset.graphql punctuation.operation.graphql
#                       ^^^^ inline.graphql apollo.commentWithTemplate meta.embedded.block.graphql meta.selectionset.graphql variable.graphql
#                           ^ inline.graphql apollo.commentWithTemplate meta.embedded.block.graphql meta.selectionset.graphql punctuation.operation.graphql
#                            ^ inline.graphql apollo.commentWithTemplate punctuation.definition.string.template.end.js
#                             ^^ inline.graphql
>/**
#^^^^ inline.graphql
> * GraphQL
#^^^^^^^^^^^ inline.graphql
> */ `query {test}`;
#^^^^^^^^^^^^^^^^^^^^ inline.graphql
>
>/* GraphQL */
#^^^^^^^^^^^^^^ inline.graphql
>`query {test}`;
#^^^^^^^^^^^^^^^^ inline.graphql
>
>const _3 = /* GraphQL */ `
#^^^^^^^^^^^ inline.graphql
#           ^^^^^^^^^^^^^ inline.graphql apollo.commentWithTemplate comment.block.js
#                        ^ inline.graphql apollo.commentWithTemplate
#                         ^ inline.graphql apollo.commentWithTemplate punctuation.definition.string.template.begin.js
>  query {
#^^ inline.graphql apollo.commentWithTemplate meta.embedded.block.graphql
#  ^^^^^ inline.graphql apollo.commentWithTemplate meta.embedded.block.graphql keyword.operation.graphql
#       ^ inline.graphql apollo.commentWithTemplate meta.embedded.block.graphql meta.selectionset.graphql
#        ^ inline.graphql apollo.commentWithTemplate meta.embedded.block.graphql meta.selectionset.graphql punctuation.operation.graphql
>    test
#^^^^ inline.graphql apollo.commentWithTemplate meta.embedded.block.graphql meta.selectionset.graphql
#    ^^^^ inline.graphql apollo.commentWithTemplate meta.embedded.block.graphql meta.selectionset.graphql variable.graphql
>  }
#^^ inline.graphql apollo.commentWithTemplate meta.embedded.block.graphql meta.selectionset.graphql
#  ^ inline.graphql apollo.commentWithTemplate meta.embedded.block.graphql meta.selectionset.graphql punctuation.operation.graphql
>`;
#^ inline.graphql apollo.commentWithTemplate punctuation.definition.string.template.end.js
# ^^ inline.graphql
>
>/* graphql */
#^^^^^^^^^^^^^^ inline.graphql
>`query {test}`;
#^^^^^^^^^^^^^^^^ inline.graphql
>
>/* gql */
#^^^^^^^^^^ inline.graphql
>`query {test}`;
#^^^^^^^^^^^^^^^^ inline.graphql
>